XSLT/tslaqdctomods.xsl

```diff
@@ -190,4 +192,19 @@
             <typeOfResource><xsl:value-of select="normalize-space(.)"/></typeOfResource>
```

Comment:
@markpbaggett - We also need to deal with capitalization here if we want typeOfResource to validate. The example record contains "Still Image" instead of "still image".

Reply:
@mlhale7, this attempts to deal with dc:language.  If this typeOfResource problem exists, I'd like to deal with it with a separate issue.  Primarily, I want to know what all values exist for this and how to approach it.

Do you have any problems with us handling this in a separate p.r. with a separate isue?  If it's super messy data, I might suggest we drop the template altogether.  I want to first find all values from them as qdc dc:types and figure out how to best approach.

Reply:
@mlhale7, I created a different issue to address this:

https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/125

Reply:
@mlhale7, I decided to go ahead and deal with the typeOfResource issue in this pull request.  The new template is based on all current values from TSLA.  If they use a new one, they won't get a typeOfResource until we update the xsl:param definition.

Reply:
@markpbaggett - thanks for this. As typeOfResource isn't a required element for DPLA, I think it's okay if there are records that don't get this information because we haven't accounted for the value they included. Currently, this is working great with the data we have.